.github/workflows/create_release.yaml

```
@@ -0,0 +1,52 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+    - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+name: Upload Release Asset
+
+jobs:
+  build:
+    name: Upload Release Asset
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: '1.16.6'
+      - name: Create infrastructure components
+        run: |
+          make create_infrastructure-components
```

You should specify the `IMG` env

---

Something like this:
```suggestion
          make create_infrastructure-components IMG="quay.io/eranco74/cluster-api-provider-agent:${{ steps.vars.outputs.sha_short }}
```